schemas/Subscription.json

```diff
@@ -22,7 +22,10 @@
     },
     "target": {
       "description": "Target callback URI",
-      "$ref": "Iri.json"
+      "oneOf": [
+        { "$ref": "Iri.json" },
+        { "type": "string", "pattern": "^([*]{1})" }
```

Comment:
I don't understand, wouldn't that pertain to "subject"? Also, is the pattern supposed to match "\*"? Wouldn't that be `^\*$`?